src/riak_pb_ts_codec.erl

```diff
@@ -131,6 +131,16 @@ decode_row([], Acc) ->
 decode_row([#tsrow{cells = Row} | T], Acc) ->
     decode_row(T, [decode_cells(Row, []) | Acc]).
 
+-spec decode_numeric(list()) -> float().
```

Comment:
Yeahhhh…

This code (all the time series protobuffs client/server stuff, not just this PR in particular) needs tests so we can actually reason about what's going on and what works and what doesn't.